py/desidatamodel/check.py

```diff
@@ -393,12 +393,16 @@ def validate_prototype(self, error=False, skip_keywords=False):
                 missing_keywords = mkw_set - dkw_set
                 extra_keywords = dkw_set - mkw_set
                 if len(missing_keywords) > 0:
-                    log.warning("File %s HDU%d missing keywords according to %s: %s",
-                        self.prototype, i, self.filename, str(missing_keywords))
+                    log.warning(
```

Comment:
The changes in this file appear to be just a change in spacing.  Is there any need for it?

Reply:
This change fixed the [Failing unit tests for code style](https://travis-ci.org/github/desihub/desidatamodel/jobs/691486131). I suppose the change is only necessary insofar as its necessary to have those unit tests.

Pedantic, I know.